src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/NewAzureStorageAccount.cs

```diff
@@ -134,6 +134,23 @@ public PSNetworkRuleSet NetworkRuleSet
             get; set;
         }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable HierarchicalNamespace for the Storage account.")]
+        [ValidateNotNullOrEmpty]
+        public bool EnableHierarchicalNamespace
```

Comment:
As mentioned in the review, this boolean was okay in Preview, but will need to be converted to either a SwitchParameter or Enum now that it is being moved to Stable.

Reply:
@maddieclayton 

We had a meeting before with feature team and Mark, feature team also prefer to use bool instead of enum. Finally we all agreed change the parameter name as following, and it's good for both preview and GA:
`-EnableHierarchicalNamespace [bool]`

You can confirm this with @markcowl

Reply:
We'll quickly go over this during the meeting at 5 today - the review was far enough back that we'd like to quickly revisit.

Reply:
We have get agreement this is good for GA in the meeting.